openassessment/locale/he/LC_MESSAGES/django.po

```diff
@@ -302,15 +305,6 @@ msgid ""
 "assessed sample responses that the instructor provides."
 msgstr "אלגוריתם מעריך את תשובות הלומדים באמצעות השוואת התשובות לתשובות ממדגם שנערך מראש ושסיפק המנחה."
 
-#: openassessment/templates/openassessmentblock/edit/oa_edit_example_based_assessment.html:16
-#: openassessment/templates/openassessmentblock/edit/oa_edit_student_training.html:18
-msgid ""
-"Enter one or more sample responses that you've created, and then specify the"
-" options that you would choose for each criterion in your rubric. Note that "
-"you must add your rubric to the Rubric tab before you can complete this "
-"step."
-msgstr "הזן תשובה לדוגמה אחת או יותר שכתבת. לאחר מכן פרט את האפשרויות שהיית בוחר לכל קריטריון בכותרת שלך. שים לב שחובה להוסיף את הכותרת שלך לכרטיסיית כותרת לפני שתוכל להשלים שלב זה."
```

Comment:
This stanza was not removed; it has just been moved [down to here by `master`](https://github.com/edx-olive-oc/edx-ora2/pull/4/files#diff-e52db3a9ff8e232a1697ce81011aa920R488).